src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Annotate.java

```diff
@@ -672,6 +672,13 @@ private Attribute getAnnotationPrimitiveValue(Type expectedElementType, JCExpres
             log.error(tree.pos(), Errors.AttributeValueMustBeConstant);
             return new Attribute.Error(expectedElementType);
         }
+
+        // Scan the annotation element value and then attribute
+        // the internal annotations to prevent the compiler from crashing
```

Comment:
I would remove the: `to prevent the compiler from crashing` part, suggestion:

```
Scan the annotation element value and then attribute nested
annotations if present
```

Reply:
Fixed.